### PR TITLE
Adds a cooldown for setting alert level through comms console

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Adds a 1 minute cooldown for changing the alert level at a comms console.

## Why It's Good For The Game
Players can currently change the alert level really really fast and it's really really annoying, borderline deafening and spamming chat.

## Testing
Tried to change the alert level, got cooled down.

## Changelog
:cl:
tweak: The communications console now has a 1m delay between changes of alert level.
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/communications.dm [hunk 1]
```diff
@@ -29,6 +29,7 @@
 
 	var/message_cooldown
 	var/centcomm_message_cooldown
+	var/alert_level_cooldown
```
Comment (Sirryan2002) [inline code/game/machinery/computer/communications.dm:32]:
```suggestion
	var/alert_level_cooldown = 0
```

**Review by Sirryan2002 — APPROVED** [review]

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]
Do Sirryan's change please

File: code/game/machinery/computer/communications.dm (hunk 1 above)
Comment (lewcc) [reply code/game/machinery/computer/communications.dm]: Addressed

**Comment by lewcc** [comment]
Changes addressed

**Pull request merged**